sdk/communication/azure-communication-phonenumbers/src/samples/java/com/azure/communication/phonenumbers/FindAndPurchasePhoneNumber.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.phonenumbers;
+
+import com.azure.communication.phonenumbers.models.PhoneNumberCapabilities;
+import com.azure.communication.phonenumbers.models.PhoneNumberCapabilityType;
+import com.azure.communication.phonenumbers.models.PhoneNumberSearchOptions;
+import com.azure.communication.phonenumbers.models.PhoneNumberSearchResult;
+import com.azure.communication.phonenumbers.models.PhoneNumberType;
+import com.azure.communication.phonenumbers.models.PhoneNumberAssignmentType;
+import com.azure.communication.phonenumbers.models.PhoneNumberOperation;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.PollResponse;
+
+public class FindAndPurchasePhoneNumber {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
+        String accessKey = System.getenv("AZURE_COMMUNICATION_KEY");
+        AzureKeyCredential keyCredential = new AzureKeyCredential(accessKey);
+
+        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
```

Comment:
Do we still want to have netty in here? I thought we removed them from the readmes

Reply:
Unresolving as I wasn't sure what the decision is here?

Reply:
removed the netty client